File: frontend/src/models/CreateClientInput.ts
export default class CreateClientInput {
    Email: string;
    Password: string;
    Name: string;
    Phone: string;

    constructor(email: string, password: string, name: string, phone: string) {
        this.Email = email;
        this.Password = password;
        this.Name = name;
        this.Phone = phone;
    }
};

File: Backend/Backend/API/Controllers/Models/CreateClientInput.cs
namespace Backend.API.Controllers.Models
{
    public class CreateClientInput
    {
        public string Name { get; set; }

        public string Email { get; set; }

        public string Password { get; set; }

        public string Phone { get; set; }

        public CreateClientInput(string name, string email, string password, string phone)
        {
            Name = name;
            Email = email;
            Password = password;
            Phone = phone;
        }
    }
}
